src/index.js

```diff
@@ -1,17 +1,51 @@
+import Weather from './Weather';
+import showWeather from './DOM';
 
+const objectBuilder = (json) => {
+  const mainJson = json.main;
+  const weather = json.weather[0].main;
+  const weatherDescription = json.weather[0].description;
+  const { temp } = mainJson;
+  const tempMin = mainJson.temp_min;
+  const tempMax = mainJson.temp_max;
+  const feelsLike = mainJson.feels_like;
+  const { humidity } = mainJson;
+  const sunRise = new Date(json.sys.sunrise * 1000);
+  const sunSet = new Date(json.sys.sunset * 1000);
+  const location = json.name;
+  return [
+    location,
+    weather,
+    weatherDescription,
+    temp,
+    tempMin,
+    tempMax,
+    feelsLike,
+    humidity,
+    sunRise,
+    sunSet,
+  ];
+};
 
-const getWeather = async city => {
-    const API_KEY = "e88a6f60008b01493b86cacd829f692b";
-    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${API_KEY}`, { mode: 'cors' })
-        .then((response) => {
-            console.log(response.json())
 
-        }).catch(function (err) {
-            console.log(err)
-        })
-}
+const getWeather = async (city) => {
+  const API_KEY = 'e88a6f60008b01493b86cacd829f692b';
+  fetch(
+    `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${API_KEY}`,
+    { mode: 'cors' },
+  )
+    .then((response) => {
+      response.json().then((response) => {
+        showWeather(new Weather(...objectBuilder(response)));
+      });
+    });
```

Comment:
- [ ] Please handle fetch errors here by chaining a `.catch` to the fetch request or wrapping the call in a try/catch block.